src/__tests__/standalone.test.js

```diff
@@ -342,7 +344,8 @@ describe("standalone", () => {
     const result = await standalone({
       files: `${fixturesGlob}/svg-icons/**/*`,
       formats: ["eot"],
-      glyphTransformFn: obj => {
+      glyphTransformFn: async obj => {
+        await wait();
         obj.unicode = ["\u0001"];
```

Comment:
Can we add new test instead modify current, i think we should support async and not async functions

Reply:
well the problem is we need an await otherwise eslint is going to complain

Reply:
We can use Promise here to avoid this, like `return Promise.resolve().then(() => glyphTransformFn(/* options */))`. It is allow to use non async function and async too

Reply:
Remove async from here too

Reply:
@Narquadah do we have any news on this part?